bim2sim/tasks/bps/bind_tz.py

```diff
@@ -17,101 +17,87 @@ class CombineThermalZones(ITask):
         * Usage
         * Window to wall ratio
     """
-
```

Comment:
please rename the file bind_tz to something more related to combining thermal zones (bind_tz sounds like you're binding the thermal zones to the building, but does not sound like aggregation).